about.html

```diff
@@ -0,0 +1,167 @@
+                dignissimos quia incidunt id pariatur odit laborum sunt
+                reprehenderit labore blanditiis temporibus modi.
+              </p>
+            </div>
+            <div class="info">
+              <h3 class="date light-black heading-medium">
+                2022.07.19(Tue) ~ 21(Thu)
+              </h3>
+              <p class="location light-black heading-small">
+                @Conference Hall, Main Campus, AIOU Islamabad
+              </p>
+            </div>
+          </div>
+        </div>
+      </header>
+    <section class="about-us flx-col-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="myLine text-center" />
+      <img src="./images/conference_photo.jpg" alt="about us community photo" class="family-photo" />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Id mollitia
+          magni enim ducimus repudiandae assumenda fugit dolorum doloribus
+          ratione dolor cum, fugiat consequatur quisquam quos deserunt amet illo
+          recusandae distinctio incidunt tenetur veritatis facilis excepturi
+          eum. Consectetur repellendus sunt, dolorum exercitationem maiores
+          optio architecto voluptates deserunt quia nemo quod magni.
+        </p>
+        <h4>
+          <br />
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Maiores ipsa
+        </h4>
+        <p>
+          <br />
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Maiores ipsa
+          ipsam explicabo eveniet iure reiciendis accusamus libero expedita ea
+          suscipit?
+        </p>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <section class="past-activities" id="past-activities">
+      <h2 class="past-heading light-black heading-medium">
+        See the past Club Activities
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <p class="past-content light-black text-center">
+        Take a look at the last two years activities which took place in
+        Istanbul
+      </p>
+      <div class="activities">
+        <div class="past-first-activity act">
+          <div class="first-activity-content flx-col-center">
+            <h3>2020</h3>
+            <p>AIOU Internatioal Summit on Technology</p>
+          </div>
+        </div>
+        <div class="past-second-activity act">
+          <div class="first-activity-content flx-col-center">
+            <h3>2019</h3>
+            <p>Robotics Conference of Automation</p>
+          </div>
+        </div>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <footer class="footer flx-col-center" id="contacts">
+        <h3 class="contacts light-black heading-medium">Contact</h3>

```

Comment:
- [ ] Kindly the Partners section exists on the mobile version kindly add it.
Original design
![image](https://user-images.githubusercontent.com/35535130/175191634-82034a68-c1c0-4f63-9e01-52e7ebdef177.png)

Your design
![image](https://user-images.githubusercontent.com/35535130/175191679-ead848de-60c1-4ea2-bc84-48402a58e584.png)